### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by diptadas**
[Worker](https://github.com/appscode/g2/blob/master/README.md#worker)
Unsolved reference "ErrHandler", it should be "ErrorHandler"
https://github.com/appscode/g2/blob/master/worker/worker.go#L37
Argument "net" missing in "AddServer"
https://github.com/appscode/g2/blob/master/worker/worker.go#L71

[Client](https://github.com/appscode/g2/blob/master/README.md#client)
"JobNormal" is in g2/pkg/runtime,  not in g2/client
https://github.com/appscode/g2/blob/master/pkg/runtime/protocol.go#L94

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,3 @@
-[![Go Report Card](https://goreportcard.com/badge/github.com/appscode/g2)](https://goreportcard.com/report/github.com/appscode/g2)
```
Comment (TamalSaha) [inline README.md]: Why are you removing links to go reportcard?

File: README.md [hunk 2]
```diff
@@ -72,44 +70,97 @@ how to change monitor address ?
 ## Worker
 
 ```go
-// Limit number of concurrent jobs execution.
-// Use worker.Unlimited (0) if you want no limitation.
-w := worker.New(worker.OneByOne)
-w.ErrHandler = func(e error) {
-	log.Println(e)
+import (
```
Comment (sadlil) [inline README.md]: This is not example. Examples are under `/examples` directory. If you need to add more example or fix those fix it there.
This is just a code sample. Stick with the format as it is, just fix things that needs to be fixed.

File: README.md [hunk 3]
```diff
@@ -72,44 +70,97 @@ how to change monitor address ?
...
-c.ErrorHandler = func(e error) {
-	log.Println(e)
-}
-echo := []byte("Hello\x00 world")
-echomsg, err := c.Echo(echo)
-// ... error handling
-log.Println(string(echomsg))
-jobHandler := func(resp *client.Response) {
-	log.Printf("%s", resp.Data)
+import (
```
Comment (sadlil) [inline README.md]: This is not example. Examples are under `/examples` directory. If you need to add more example or fix those fix it there.
This is just a code sample. Stick with the format as it is, just fix things that needs to be fixed.

File: README.md (hunk 1 above)
Comment (sadlil) [inline README.md]: @diptadas Probably you are behind master.

File: README.md [hunk 4]
```diff
@@ -5,17 +5,19 @@
...
 
 	"github.com/appscode/g2/client"
 
-The worker package will help developers in developing Gearman worker
-service easily.
+The worker package will help developers in developing Gearman worker service easily.
 
 	"github.com/appscode/g2/worker"
+	    
+The gearadmin package is used to implement a Gearman admin client.
```
Comment (TamalSaha) [inline README.md]: Change it to:

The `gearadmin` package implements a client for Gearman admin protocol.

**Pull request merged**